miscellaneous.py

```diff
@@ -0,0 +1,78 @@
+import modules.fastq_filter_functions.py
+import modules.nucleic_acid_module.py
+import modules.protein_module.py
+import os
+
```

Comment:
```suggestion
import os

import modules.fastq_filter_functions
import modules.nucleic_acid_module
import modules.protein_module


```

По PEP8, после импортов должны идти две пустые строки. Сначала импортируются встроенные библиотеки, потом уже свои, между ними одна пустая строка. Свои скрипты с расширением не импортируются - очень советую тестировать свой код перед тем, как отправлять)